Library/Homebrew/formula.rb

```diff
@@ -2933,7 +2933,8 @@ def pour_bottle?(only_if: nil, &block)
         lambda do |_|
           on_macos do
             T.cast(self, PourBottleCheck).reason(+<<~EOS)
-              The bottle needs the Apple Command Line Tools to be installed.
+              The bottle needs the standalone Apple Command Line Tools to be installed.
+              The full version of Xcode is not compatible with this bottle.
```

Comment:
I wonder if we can/should make this part of the message conditional on whether you have Xcode installed. (I think we can check this using `MacOS::Xcode.installed?`.)

Reply:
I added a conditional check.